README.md

```diff
@@ -8,7 +8,7 @@ This project contains code and documentation for the LENSS-Sensor being develope
      * serverconfig.ini: the standard serverconfig file holds port and filename data for both the arduino and SQM-LU, has a dummy line of data used when experimenting with how the pi's programs affect data, and directs to a log monitoring the status of connected devices
     * There are additional serverconfigs for certain sensors or people such as serverconfigMGB.ini (Marc's serverconfig file)
   * arduino: code used for the operation of sensors via arduino:
-    * TSL237_Uno.ino: the current program used for the arduino in Sensor0001; when dark enough, it interrupts all current processes to update a count when light hits the tsl and prints luminosity alongside temperature data 
+    * TSL237_Uno.ino: the current program used for the arduino in Sensor0001; when dark enough, it interrupts all current processes to update a count when light hits the tsl and prints luminosity alongside temperature data
```

Comment:
Took the space removal from this PR: https://github.com/yerkesobservatory/lenss-sensor/pull/3